match/bossier_city_pd.py

```diff
@@ -1,9 +1,32 @@
-from datamatch import JaroWinklerSimilarity, ThresholdMatcher, ColumnsIndex
+from datamatch import JaroWinklerSimilarity, ThresholdMatcher, ColumnsIndex, Swap
```

Comment:
Doesn't look like you're using `Swap` in this script.

Reply:
Thank you.